src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/TableBuilderDataService.cs

```diff
@@ -42,6 +43,23 @@ public override ResultViewModel Query(ObservationQueryContext queryContext)
                     _resultBuilder.BuildResult(observation, queryContext.Indicators))
             };
         }
+        
+        public override async Task<ResultViewModel> QueryAsync(ObservationQueryContext queryContext)
+        {
+            var observations = GetObservations(queryContext).ToList();
```

Comment:
This doesn't seem right. Aren't you duplicating exactly the same code as the `Query` method? This adds the async keyword but there are no awaits.

Reply:
I've amended this to just call the none async method, rather than duplicate. This allows us to call the query with an await in another service (as done in this PR) even though the query method its self isnt async.